tests/integration/telemetry/stackdriver/testdata/metadata_exchange_filter.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
```

Comment:
we have this file in like 10 different places. Can we reuse it from one of those places instead of copying it again? Its never kept in sync

Reply:
I was trying run test with istioctl, but blocked by the installe bug (https://github.com/istio/installer/commit/2d8249df15868ab1bdd116d5368285613a4fd623). The fix has not reached istio/istio. I am planning to migrate both sd and prom test to istioctl once that lands. If you feel uncomfortable with the duplication here, I could update operator first.

Reply:
I am fine with merging this in as is if we have a plan to switch it over in the near future. We already do it for prom filter anyways. I am working on updating operator right now

Reply:
Ok cool, let's hold this pr and wait for operator to be updated.